cvat/apps/engine/migrations/0085_segment_chunks_updated_date.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.2.15 on 2024-09-25 13:52
+
+from datetime import datetime
+from django.db import migrations, models
+
+
+def _get_0083_move_to_segment_chunks_migration_date(apps, schema_editor) -> datetime:
+    with schema_editor.connection.cursor() as cursor:
+        cursor.execute("""\
+            SELECT applied
+            FROM django_migrations
+            WHERE app = %s AND name = %s
+        """, ['engine', '0083_move_to_segment_chunks'])
+        return cursor.fetchone()[0]
+
+
+def init_chunks_updated_date(apps, schema_editor):
+    # The 0083 migration changed data distribution by chunks
+    migration_0083_date = _get_0083_move_to_segment_chunks_migration_date(apps, schema_editor)
```

Comment:
Why not use current datetime?

Reply:
Or just `task_created_date` as done below

Reply:
Because this will be incorrect. 
If the task was created before the migration, it's chunks were updated when the migration was applied.
If the task was created after the migration, it's chunks were updated at the moment of task creation.
Neither of this is `now`.

Reply:
It is correct enough, as the feature to upgrade chunks (shuffle honeypots) was not merged yet. 

> chunks were updated when the migration was applied

How exactly these chunks were updated in the migration? They only were re-numerated.

Reply:
> How exactly these chunks were updated in the migration? They only were re-numerated.

No, the chunks on the end of the segment could be truncated, and for videos with overlap there were entirely new chunks.